### PR TITLE
Ux/consolidate space mgmt

### PR DESCRIPTION
**Pull request by osala-eng**
## Description

Implements #52

Updated admin docs ui layout
admin_docs

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor and code improvement (non-breaking change which improves code quality and/or performance)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation
- [ ] Tests
- [ ] Other chores such as maintenance

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration by modifying the list below.

- [ ] Test A
- [ ] Test B
- [ ] Test C

**Test Configuration**:

Please describe the test setup. List them below as bullet points.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] The commit message follows the convention of this project

### CODE REVIEW

**Comment by janaka** [comment]
@osala-eng I added a comment to the issue re: naming the new section "Admin Spaces"

**Comment by osala-eng** [comment]
Renamed consolidated space to `Admin Docs`

image

File: web/utils/layout.py [hunk 1]
```diff
@@ -557,52 +564,63 @@ def _render_view_space_details_with_container(
...
-                disabled=True,
-                format_func=lambda x: x[1],
-            )
-            _render_space_data_source_config_input_fields(data_source, f"update_space_details_{id_}_", ds_configs)
-            st.form_submit_button("Save", on_click=handle_update_space_details, args=(id_,))
+        _edit_space_details_form(space_data, data_source)
 
 
-def _render_manage_space_permissions(space_data: Tuple) -> None:
+def _manage_space_permissions_form(space_data: Tuple) -> None:
```
Comment (janaka) [inline web/utils/layout.py:592]: ideally these functions should following `_render_*` prefix naming convention.

File: web/utils/layout.py [hunk 2]
```diff
@@ -557,52 +564,63 @@ def _render_view_space_details_with_container(
...
+            "Groups",
+            options=[(g[0], g[1]) for g in list_user_groups()],
+            default=permissions[SpaceAccessType.GROUP],
+            key=f"manage_space_permissions_{id_}_{SpaceAccessType.GROUP.name}",
+            format_func=lambda x: x[1],
+        )
+        st.form_submit_button("Save", on_click=handle_manage_space_permissions, args=(id_,))
+
+
+def _render_manage_space_permissions(space_data: Tuple) -> None:
```
Comment (janaka) [inline web/utils/layout.py:619]: this function now seems a bit off.

**Review by janaka — COMMENTED** [review]
Left a few comments

File: web/utils/layout.py (hunk 2 above)
Comment (osala-eng) [reply web/utils/layout.py:635]: I separated the forms from original funtions to be able to render them directly into the consolidated space without having to click the button as below, while leaving the original functionality as it was.
image

**Comment by osala-eng** [comment]
> Left a few comments

I've rebased this branch

I rearranged the `Load chat history` and `New Chat` buttons to be on a single collumn, hope that's okay.

image

**Review by janaka — APPROVED** [review]

**Pull request merged**